mmf/datasets/builders/localized_narratives/masked_dataset.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+from abc import ABC
+from typing import Type, Union
+
+import torch
+from mmf.common.sample import Sample
+from mmf.common.typings import MMFDatasetConfigType
+from mmf.datasets.builders.localized_narratives.database import (
+    LocalizedNarrativesAnnotationDatabase,
+)
+from mmf.datasets.mmf_dataset import MMFDataset
+
+
+class MaskedLocalizedNarrativesDatasetMixin(ABC):
+    def build_annotation_db(self) -> Type[LocalizedNarrativesAnnotationDatabase]:
```

Comment:
This should not use `Type`. We need to fix this at other places in MMF as well. But this would be just 
```suggestion
    def build_annotation_db(self) -> LocalizedNarrativesAnnotationDatabase:
```